tutorial/out-of-game/auth.md

```diff
@@ -4,6 +4,6 @@ The first page you're greeted with when you visit PlanarAlly is the Login/Regist
 
 ![](./login.png)
 
-Currently no email address is required for sign-up, which means that there is no solid recovery mechanism in case you lose your password, so make sure you don't lose it ;)
+Currently no email address is required for sign-up, which means that there is no solid recovery mechanism in case you lose your password, so make sure you don't lose it ;) that said it is technically possible for the passwords to be changed by a [server administrator](../../docs/server/management.md). That said there is no automated system so your results may vary.
```

Comment:
```suggestion
Currently no email address is required for sign-up, which means that there is no solid recovery mechanism in case you lose your password, so make sure you don't lose it ;) that said it is technically possible for the passwords to be changed by a [server administrator](/docs/server/management). That said there is no automated system so your results may vary.
```